backend/src/main/java/se/chalmers/dat265/group1/api/physical/PhysicalAPI.java

```diff
@@ -2,7 +2,11 @@
 
 import se.chalmers.dat265.group1.model.PhysicalPart;
 import se.chalmers.dat265.group1.model.PhysicalPrint;
+import se.chalmers.dat265.group1.model.SlmData;
+import se.chalmers.dat265.group1.model.StlData;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'se.chalmers.dat265.group1.model.StlData'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)